pkg/networkservice/chains/nsmgr/server.go

```diff
@@ -116,7 +116,7 @@ func NewServer(ctx context.Context, nsmRegistration *registryapi.NetworkServiceE
 		recvfd.NewServer(), // Receive any files passed
 		interpose.NewServer(&interposeRegistryServer),
 		filtermechanisms.NewServer(&urlsRegistryServer),
-		connect.NewServer(ctx,
+		connect.NewServer(ctx, 100*time.Millisecond,
```

Comment:
I feel this should be an optional parameter.

Reply:
Actually `grpc.WithTimeout` option serves exactly the purpose we need but it is deprecated :(
So OK, now there are `connect.WithDialTimeout` and `connect.WithDialOptions` :)